### ISSUE TITLE
Not in scope: `defaultTimeLocale'

### ISSUE THREAD

**Issue by twsh** [nfjinjing/hack2-contrib#4]
I'm trying to install hack2-contrib as part of installing maid. After

```
cabal update
cabal install
```

I get the following error

```
src/Hack2/Contrib/AirBackports.hs:31:26:
    Not in scope: `defaultTimeLocale'

src/Hack2/Contrib/AirBackports.hs:40:23:
    Not in scope: `defaultTimeLocale'
Failed to install hack2-contrib-2015.5.4
cabal: Error: some packages failed to install:
hack2-contrib-2015.5.4 failed during the building phase. The exception was:
ExitFailure 1
maid-2014.10.14 depends on hack2-contrib-2015.5.4 which failed to install.
```

I'm using OS X 10.10.5 if that matters.

**Comment by nfjinjing** [comment]
only runs on GHC 7.10 +
